lesson14/main.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+const wrapper = document.getElementById("wrapper");
+const lists = document.getElementById("lists");
+const createElBtn = document.getElementById("create-element-btn");
+createElBtn.classList.add("btn");
+const modal = document.getElementById("modal");
+const displayModal = document.getElementById("display-modal");
```

Comment:
IMO
I think it's better to add "js-" to the html id.
[JavaScriptと紐づく要素はjs-の接頭辞]
(https://zenn.dev/haniwaman/articles/bf392f397c8db7341881#javascript%E3%81%A8%E7%B4%90%E3%81%A5%E3%81%8F%E8%A6%81%E7%B4%A0%E3%81%AFjs-%E3%81%AE%E6%8E%A5%E9%A0%AD%E8%BE%9E)

I learned it from Morita-san too, and I think `ul` is appropriate here.
Because `ul` is a list and `li` is a listitem.
If you use lists, it will look as if there are several `ul`.
[リスト系のコンポーネント名 Images と ImageList はどっちが有利か](https://qiita.com/misuken/items/d62c304fd6d730801ddb)
 
```suggestion
const wrapper = document.getElementById("js-wrapper");
const ul = document.getElementById("js-ul");
const modalBtn = document.getElementById("js-modalBtn");
modalBtn.classList.add("btn");
const modal = document.getElementById("js-modal");
const openModalBtn = document.getElementById("js-openModalBtn");
```

Also, there was a part of the constant naming that bothered me.

```
create-element-btn
``` 
This is `js-modalBtn`


and

```
display-modal
```
This is `js-openModalBtn`

Reply:
Thank you for comment.
I changed the name of constants and id. Please check them.
6b9cd79